### PR TITLE
feat: [M3-8017] - Support Linodes in Distributed Compute Regions on Image Create 

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝

- Shows a notice saying `This Linode is in a distributed compute region. Images captured from this Linode will be stored in the closest core site.` if you select a Linode that is in a distributed region on the Image Create page ℹ️ 
- Shows helper text saying `Image will be stored in the closest core site to (us-den-edge-1)` below the Linode select if the selected Linode is in a distributed region ℹ️ 

These changes are based on the Image Service Gen2 Figma wireframes 🎨  - Ask me for a link if you want to reference it.

## Preview 📷

| Before  | After   |
| ------- | ------- |
| Screenshot 2024-06-04 at 1 53 39 PM | Screenshot 2024-06-04 at 1 52 41 PM |

### Prerequisites
- Have a Linode in a distributed region on your account

### Verification steps
- Go to the Image Create page (http://localhost:3000/images/create/disk)
- Select a Linode that is in a distributed region
- Verify you see the notice and helper text that UX reflects in the mockups 👁️🎨 

## As an Author I have considered 🤔

- [x] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [x] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/src/features/Images/ImagesCreate/CreateImageTab.test.tsx [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+    const submitButton = getByText('Create Image').closest('button');
+
+    await userEvent.click(submitButton!);
+
+    // Verify success toast shows
+    await findByText('Image scheduled for creation.');
+  });
+
+  it('should render a notice if the user selects a Linode in a distributed compute region', async () => {
```
Comment (bnussman-akamai) [inline packages/manager/src/features/Images/ImagesCreate/CreateImageTab.test.tsx:105]: Unit test for this PR's addition

File: packages/manager/src/features/Images/ImagesCreate/CreateImageTab.test.tsx [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+      'This Linode is in a distributed compute region. Images captured from this Linode will be stored in the closest core site.'
+    );
+
+    // Verify helper text renders
+    await findByText(
+      `Image will be stored in the closest core site to (${linode.region})`
+    );
+  });
+
+  it('should render an encryption notice if disk encryption is enabled and the Linode is not in a distributed compute region', async () => {
```
Comment (bnussman-akamai) [inline packages/manager/src/features/Images/ImagesCreate/CreateImageTab.test.tsx:142]: Unit test for @dwiley-akamai 's change in https://github.com/linode/manager/pull/10521

Comment (bnussman-akamai) [reply packages/manager/src/features/Images/ImagesCreate/CreateImageTab.test.tsx:142]: Oops, I just realized you covered this in a cypress test. I'll keep it here anyway unless there are any objections.

File: packages/manager/src/features/Images/ImagesCreate/CreateImageTab.tsx [hunk 3]
```diff
@@ -153,6 +174,12 @@ export const CreateImageTab = () => {
               created from a raw disk or a disk that&rsquo;s formatted using a
               custom file system.
             </Typography>
+            {linodeIsInDistributedRegion && (
+              <Notice variant="info">
+                This Linode is in a distributed compute region. Images captured
+                from this Linode will be stored in the closest core site.
+              </Notice>
+            )}
```
Comment (hkhalil-akamai) [inline packages/manager/src/features/Images/ImagesCreate/CreateImageTab.tsx:182]: Can we increase the margins above and below this notice to match the mockups?

|This PR|Figma mockup|
|-|-|
|Screenshot 2024-06-06 at 5 02 30 PM|Screenshot 2024-06-06 at 5 02 07 PM|

File: packages/manager/src/features/Images/ImagesCreate/CreateImageTab.test.tsx
Comment (hkhalil-akamai) [inline packages/manager/src/features/Images/ImagesCreate/CreateImageTab.test.tsx:1]: Thanks for adding unit tests for this component! 🙌🏽

**Review by hkhalil-akamai — APPROVED** [review]
Verified the notice appears when selecting a Linode in a distributed region.

File: packages/manager/src/features/Images/ImagesCreate/CreateImageTab.tsx (hunk 3 above)
Comment (bnussman-akamai) [reply packages/manager/src/features/Images/ImagesCreate/CreateImageTab.tsx:182]: I increased overall spacing in ba513a7, let me know if that looks okay to you! 👁️

Comment (hkhalil-akamai) [reply packages/manager/src/features/Images/ImagesCreate/CreateImageTab.tsx:182]: Looks good, thanks!

**Review by cpathipa — APPROVED** [review]

**Pull request merged**